cs-engine/1.13/index.md

```diff
@@ -2,6 +2,11 @@
 title: Install CS Docker Engine 1.13
 description: Learn how to install the commercially supported version of Docker Engine.
 keywords: docker, engine, install
+redirect_from:
```

Comment:
I think you forgot to commit the `cs-engine/1.12` file. This way you'll have the same redirects on both articles.